terraform-example/README.md

```diff
@@ -0,0 +1,47 @@
+## This example and documentation are for the Beta v.1 of the Terraform Repositories, available exclusively to JFrog Beta customers.
+
+# Terraform Example
+
+## Overview
+This example demonstrates how to publish terraform modules to Artifactory.
+
+## Before Running the Example
+### Set Up the Environment
+1. Create local repository in Artifactory.
```

Comment:
Create local repository in Artifactory.
-->
Create a local Terraform repository in Artifactory.